src/web/client/extension.ts

```diff
@@ -4,14 +4,24 @@
  * ------------------------------------------------------------------------------------------ */
 
 import * as vscode from "vscode";
+import TelemetryReporter from "@vscode/extension-telemetry";
+import { AI_KEY } from '../../client/constants';
+
+let _telemetry: TelemetryReporter;
 
 export function activate(context: vscode.ExtensionContext): void {
     console.log("Activated web extension!"); // sample code for testing the webExtension
+    // setup telemetry
+    _telemetry = new TelemetryReporter(context.extension.id, context.extension.packageJSON.version, AI_KEY);
+    context.subscriptions.push(_telemetry);
+    _telemetry.sendTelemetryEvent("Start");
+    _telemetry.sendTelemetryEvent("activated");
```

Comment:
this line should be the last line of the function. as it is in the other extensions.ts